### ISSUE TITLE
Utility to generate an environment map from a scene

### ISSUE THREAD

**Issue by almarklein** [pygfx/pygfx#338]
IIRC @panxinmiao proposed this earlier. Then we can create stuff like [this](https://threejs.org/examples/?q=dynamic#webgl_materials_cubemap_dynamic).

**Comment by panxinmiao** [comment]
Yes, we also need a corresponding PMREM (Prefiltered, Mipmapped Radiance Environment Map) generation utility to provide physically-correct scene based lighting.
